example/prometheus-operator-crd/prometheus.crd.yaml

```diff
@@ -2831,6 +2831,10 @@ spec:
                 honor_labels. If HonorLabels is set in ServiceMonitor or PodMonitor
                 to true, this overrides honor_labels to false.
               type: boolean
+            overrideHonorTimestamps:
+              description: HonorTimestamps configures Prometheus whether it should
```

Comment:
`overrideHonorTimestamps` in description?

Also can we elaborate more? This flag allows to set honor_timestamp per ServiceMonitor right? Can put this into description? (: